README.md

```diff
@@ -87,142 +87,29 @@ Check the below video for a quick demo of XC3.
-
-    And run the following command
-
-    ```
-    terraform init
-    ```
-
-    `Expected Output: It will create .terraform directory in xc3/infrastructure/  location
-        Initializing modules...
-        - infrastructure in modules/networking
-        - infrastructure in modules/xc3
-        * provider.aws: version = "~> 4.0."
-        Terraform has been successfully initialized!
-`
-
-6.  Run planner command under `XC3/infrastructure` directory.
-
-    ```bash
-    terraform  plan -var-file=input.tfvars
-    ```
-
-        ```bash
-        This command will generate a preview of all the actions which terraform is going to execute.
-            Expected Output: This command will be giving output something like below
-                    Plan: 20 to add, 0 to change, 0 to destroy.
-                    ------------------------------------------------------------------------
-        ```
-
-7.  Run actual Apply command under `xc3/infrastructure` directory to deploy all the resources into AWS master account.
-    This step may take `10-15` mins.
-
-    ```bash
-    terraform apply -var-file=input.tfvars
-    ```
-
-    The output will look like below
-
-    ````bash
-        Expected output: It will ask for approval like below
-            Do you want to perform these actions?
-            Terraform will perform the actions described above.
-            Only 'yes' will be accepted to approve.
-            Enter a value:
-        ```
-    Please type "yes" and enter
-    It provides the next steps to perform
-
-    ```bash
-    Apply complete! Resources: 20 added, 0 changed, 0 destroyed.
-
-    Outputs:
-    ````
-
-8.  Please copy msg_templates in custodian directory on deployed EC2 instance
-
-    ```
-    sudo scp -r -i "keypair.pem" keypair.pem user@bastion-host-DNS:/directory-to-copy-keypair
-
-    sudo scp -r -i "keypair.pem" ../cloud_custodian_policies/  user@bastion-host-DNS:~/.
-
-    ssh -i "keypair.pem" user@bastion-host-DNS
-
-    cd directory (where keypair copied in above command)
-
-    sudo scp -r -i "keypair.pem" cloud_custodian_policies/ user@private-ip-ec2:~/.
-
-    ssh -i "keypair.pem" user@private-ip-ec2
-
-    sudo chown root:root /home/ubuntu/cloud_custodian_policies/msg_templates/*.html.j2
-
-    sudo cp /home/ubuntu/cloud_custodian_policies/msg_templates/*.html.j2 /custodian/lib/python3.8/site-packages/c7n_mailer/msg-templates/
-
-    ```
-
-9.  Please run the following steps on deployed EC2 instance to trigger XC3 lambda functions.
-
-    ```
-        source /custodian/bin/activate
-        cd cloud_custodian_policies
-
-     1. custodian run -s s3://${bucket_name}/iam-user --region ${aws_region} iam-user.yml
-
-     2. custodian run -s s3://${bucket_name}/iam-role/ --region ${aws_region} iam-role.yml
-
-     3. custodian run -s tagging-compliance --region ${aws_region} eks-tagging.yml --region all
+    cd xc3/
 
+        Note :
+            - Configure the input.sh file before running the below mentioned command:
```

Comment:
@wahajXgrid Mention the directory, it would be good if you can provide an example, what kind of configurations are required in init.sh file/

Reply:
Done